packaging/windows/tests/enterprise-detached-helper-smoke.ps1

```
@@ -0,0 +1,408 @@
+    $stopwatch = [Diagnostics.Stopwatch]::StartNew()
+    # Sequential ReadToEnd on stdout+stderr can deadlock: if the child writes
+    # more than the pipe buffer to stderr (a PowerShell error record with
+    # its stack trace easily exceeds it) while the parent is blocked on
+    # stdout, the child blocks on its stderr write and neither side makes
+    # progress. Read stderr asynchronously so both drains run concurrently.
+    $stderrReadTask = $nestedProcess.StandardError.ReadToEndAsync()
+    $capturedOutput = $nestedProcess.StandardOutput.ReadToEnd()
+    # Bound the wait so a hung child fails the smoke rather than the CI job.
+    $detachedHelperWaitBudgetMs = 5 * 60 * 1000
+    if (-not $nestedProcess.WaitForExit($detachedHelperWaitBudgetMs)) {
+        try { $nestedProcess.Kill($true) } catch { }
```

PSScriptAnalyzer (1.25.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)